lib/open_api_spex/plug/swagger_ui.ex

```diff
@@ -193,7 +193,7 @@ defmodule OpenApiSpex.Plug.SwaggerUI do
   end
 
   defp supplement_config(%{oauth2_redirect_url: {:endpoint_url, path}} = config, conn) do
-    endpoint_module = apply(Phoenix.Controller, :endpoint_module, [conn])
+    endpoint_module = Phoenix.Controller.endpoint_module(conn)
```

Comment:
Does this compile in `prod` env? OpenApiSpex should work in plain plug projects that don't use phoenix.